### PR TITLE
#184581169 commit rimraf

### PR DESCRIPTION
**Pull request by Rwibutso**
#### What does this PR do?
This PR to check if rimraf can work during deployiment

#### Description of Task to be completed?
- introduce rimraf in pachage.json as dependency

#### How should this be manually tested?
- Clone this repo git clone https://github.com/atlp-rwanda/e-comm-team-amigos-bn
- Switch your working directory cd e-comm-team-amigos-bn
- Create an .env file and update it with .env-example
- Install all the dependencies by running npm install
- Then run the project by typing npm start.

#### Any background context you want to provide?
- Have Node Js installed in your local machine
- Have rimraf installed 

#### What are the relevant pivotal tracker/Trello stories?

Story ID: #184581169

### CODE REVIEW

**Review by Mystica52 — COMMENTED** [review]
delete that package-lock.json so that the conflict removed

**Review by Emile-SHUMBUSHO — APPROVED** [review]

**Review by nshutijonathan — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "@babel/core": "^7.21.0",
     "@babel/node": "^7.20.7",
     "@babel/plugin-transform-runtime": "^7.21.0",
+    "@babel/plugin-transform-runtime": "^7.21.0",
```
Comment (Habinezajanvier) [inline package.json]: Please remove one package since they are the same

**Review by Habinezajanvier — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (nshutijonathan) [reply package.json]: you do `npm uninstall @babel/plugin-transform-runtime`  
check whether `package.json` remains with one package and then push . N.B. make sure you have one commit.

**Review by nshutijonathan — CHANGES_REQUESTED** [review]

**Review by Habinezajanvier — APPROVED** [review]
👍

**Review by wilbrord2 — APPROVED** [review]

**Pull request merged**